exercises/finale-workflow/workflow.py

```diff
@@ -0,0 +1,13 @@
+from datetime import timedelta
+from temporalio import workflow
+
+
+@workflow.defn
+class CertificateGeneratorWorkflow:
+    @workflow.run
+    async def run(self, name: str) -> str:
+        out_file_path = await workflow.execute_activity(
+            "CreatePdf", name, start_to_close_timeout=timedelta(seconds=5)
```

Comment:
If you wanted to go all out, you could make a dummy activity or protocol to represent the types of the Java side here

Reply:
I think that's a good idea to dive into in a later course